ci/helm/environments/cluster.yaml

```diff
@@ -1,3 +1,7 @@
+nuxeo:
+  extraValuesFiles:
+  - values/values-nuxeo-cluster.yaml.gotmpl
+  - values/values-nuxeo-third-party.yaml.gotmpl
```

Comment:
Should `extraValuesFiles` be converted to `valuesFiles` and move all values declaration from helmfile.yaml to environments?